### PR TITLE
Multiple wip branches

### PR DESCRIPTION
**Pull request by gregorriegler**
This is going in the suggested direction.
What do you think?

### CODE REVIEW

**Comment by simonharrer** [comment]
Goes in the right direction. The warning should only be displayed if it would *create* the new wip branch. Joining an existing wip branch should always work without a warning. The code should reflect that.

The warning itself assumes that the user might perhaps wanted to use a branch modifier. The warning message should reflect that, I think, so the user can fix it if necessary.

There might be another case as well, namely, the inverse: there exists a qualified wip branch (mob/main-green) and one issues `mob start` . There, a message asking the user that perhaps they meant `mob start --branch green` was the one the user wanted.

**Comment by gregorriegler** [comment]
I changed it so that it only shows the warning if the WIP branch does not exist yet.

Would prefer to have the inverse case handled in another PR, what do you think?

File: mob.go [hunk 1]
```diff
@@ -603,11 +602,19 @@ func sayUnstagedChangesInfo() {
 	}
 }
 
-func hasQualifiedBranches(currentBaseBranch string, remoteBranches []string, configuration Configuration) bool {
```
Comment (simonharrer) [inline mob.go:606]: getQualifiedWipBranches

File: mob.go [hunk 2]
```diff
@@ -563,6 +554,14 @@ func start(configuration Configuration) error {
 		return errors.New("cannot start; unpushed changes on base branch must be pushed upstream")
 	}
 
+	wipBranchesWithQualifier := getQualifiedBranches(currentBaseBranch, configuration)
+	if !isMobProgramming(configuration) && !hasRemoteBranch(currentWipBranch, configuration) && len(wipBranchesWithQualifier) > 0 && !configuration.WipBranchQualifierSet {
+		sayWithPrefix("other qualified mob branches detected:", " ⚠ ")
```
Comment (simonharrer) [inline mob.go:559]: Explain to user why this warning is shown.

**Comment by simonharrer** [comment]
Please a separate PR for the other change.

Can you add a changelog entry, please

**Comment by gregorriegler** [comment]
improved warnings, added changelog, and extracted sayWarning.

@simonharrer Feel free to improve on the wordings.

**Pull request merged**